blueprints/cloud-operations/network-dashboard/src/plugins/discover-cai.py

```diff
@@ -82,10 +84,16 @@ def _handle_resource(resources, asset_type, data):
   # e.g. assetType = GlobalAddress but discoveryName = Address
   resource_name = NAMES[asset_type]
   resource = {
-      'id': attrs.get('id'),
-      'name': attrs['name'],
-      'self_link': _self_link(attrs['selfLink']),
-      'assetType': asset_type
+      'id':
+          attrs.get('id'),
+      'name':
+          attrs['name'],
+      # Some resources (ex: Filestore) don't have a self_link, using parent + name in that case
+      'self_link':
+          data['parent'] + '/' + attrs['name']
```

Comment:
can you use an f-string instead of adding strings? something like `f'{data["parent"]}/{attrs["name"]}'`

Reply:
do we really need the filestore self link anywhere? otherwise I would just ignore it